### DISCUSSION TITLE
Support for 1080p and 4k screens

### DISCUSSION THREAD

**Discussion by JayeshVP24** [tailwindlabs/tailwindcss#5353] [Ideas]
Create proper utility classes for screen widths 1920px, 3840px and create all text sizes, width sizes accordingly. 
I've been using tailwind css for a long time and its exhausting to type all the custom sizes everytime. 

1080p and 4k screens are more common than one might think.

Please consider doing so.

**Comment by NovusOpus** [comment]
Can you provide an example or your specific use case? I mean, I'm pretty sure Tailwinds already covers the utility classes to scale text or other elements to be fit with most common screen sizes, unless you have some very specific use case or working with odds sizes tailwind should work fine.

Has been using Tailwind for responsive stuff from mobile to 4k or even 8k sizes, and I don't see any problem with current responsive utilities.

Not sure what your specific problems would be, so maybe providing an example will help to make it more clear?

**Comment by legoesbenr** [comment]
Many screens are above 1536px which is 2xl atm. A 4K display is 3840px same is all decent monitors.
Our use case would be that with a 3840px monitor i.e. the responsiveness is impossible to control as Tailwind's 2xl is less than half of the screen width.

It would make perfectly sense to extend tailWind with these:

'3xl': '1920px',
'4xl': '2560px',
'5xl': '3840px'
